integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/context/TestContext.java

```diff
@@ -200,6 +206,19 @@ public TestContext as(Actor actor) {
         return this;
     }
 
+    public TestContext withDescription(TestCaseDescription testCaseDesription) {
```

Comment:
`with` is annoying, because usually means one element in the background, but here we append a map. So i suggest to use a better name.